setup.cfg

```diff
@@ -159,3 +159,7 @@ max-line-length = 88
 per-file-ignores =
   docs/conf.py: D
   src/molecule/test/*: D100,D103,D104
+
+[doc8]
+# see https://github.com/PyCQA/doc8/issues/75
+ignore-path = .eggs,.tox,constraints.txt,build,molecule.egg-info,src/molecule.egg-info
```

Comment:
1) I'd multiline this
2) Isn't there a better config then package metadata?

Reply:
No multiline support, probably my comment should give you a hint that there is no much better place. FYI, flake is still inside setup.cfg for the same reasons.